documentation/source/users/rmg/installation/index.rst

```diff
@@ -40,7 +40,7 @@ For users unfamiliar with bash or Linux, we recommend looking at
 `Must Know Linux/Unix Commands <https://www.guru99.com/must-know-linux-commands.html>`_.
 
 .. NOTE::
-    The docker instructions above which specify ``rmg:3.2.0`` are for the version 3.2.0 release of RMG-Py (August 2023), the user guide for which is archived in a `4mb PDF here <https://github.com/ReactionMechanismGenerator/RMG-Py/blob/3.2.0/documentation/RMG-Py_and_Arkane_Documentation.pdf?raw=true>`_.
+    The docker instructions above which specify ``rmg:3.3.0`` are for the version 3.3.0 release of RMG-Py (July 2025), the user guide for which is archived in a `4mb PDF here <https://github.com/ReactionMechanismGenerator/RMG-Py/blob/3.3.0/documentation/RMG-Py_and_Arkane_Documentation.pdf?raw=true>`_.
```

Comment:
I think we need to build the latex pdf documentation and add to the git repo, before the commit that will be tagged as 3.3.0, which probably means before merging this PR?